templates/javascript-ethers/package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@api3/airnode-abi": "^0.3.1",
     "@api3/airnode-protocol": "^0.3.1",
+    "@api3/services": "api3dao/services#4cb79ffc406c4451c84accf42f99e6a1251a5799",
```

Comment:
Managing dependencies (and their versions) in the templates is going to be a pain. But let's think about that some other time.